src/main/java/org/openstreetmap/atlas/geography/Rectangle.java

```diff
@@ -350,10 +347,49 @@ public Rectangle expandHorizontally(final Distance distance)
      */
     public Rectangle expandVertically(final Distance distance)
```

Comment:
I think this fix should also be applied to the expandHorizontally function, so it stops at the antimeridian? Or is that already the case?